src/website2/src/app/layout.tsx

```
@@ -108,6 +122,12 @@ export default async function RootLayout({
         <meta name="twitter:description" content={description} />
         <meta name="twitter:image" content={`${siteUrl}icon.png`} />
 
+        {/* Structured data */}
+        <script
+          type="application/ld+json"
+          dangerouslySetInnerHTML={{ __html: JSON.stringify(structuredData) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)